src/bma250e/javaupm_bma250e.i

```diff
@@ -1,10 +1,25 @@
 %module javaupm_bma250e
 %include "../upm.i"
 %include "typemaps.i"
-%include "../upm_vectortypes.i"
+%include "../upm_stdvector.i"
 
 %ignore getAccelerometer(float *, float *, float *);
 
+%typemap(javaimports) SWIGTYPE %{
+import java.util.AbstractList;
+import java.lang.Float;
+%}
+
+%typemap(javaout) upm::BMA250E {
+    return new $&javaclassname($jnicall, true);
+}
+%typemap(javaout) std::vector<float> {
+    return (AbstractList<Float>)(new $&javaclassname($jnicall, true));
+}
+%typemap(jstype) std::vector<float> "AbstractList<Float>"
+
+%template(floatVector) std::vector<float>;
+
```

Comment:
That sounds good - I just want to make it as "easy" as possible going forward to support this properly in new Java examples.